presto-main/src/main/java/com/facebook/presto/server/QueryStateInfoResource.java

```diff
@@ -106,8 +109,14 @@ public void getQueryStateInfos(
             List<BasicQueryInfo> queryInfos = dispatchManager.getQueries();
             Optional<Pattern> userPattern = isNullOrEmpty(user) ? Optional.empty() : Optional.of(Pattern.compile(user));
 
+            final QueryState expectedQueryState = stateFilter == null ? null : QueryState.valueOf(stateFilter.toUpperCase(Locale.ENGLISH));
```

Comment:
maybe throw a user friendly exception if the stateFilter is not a valid queryState

Reply:
> maybe throw a user friendly exception if the stateFilter is not a valid queryState
Done.